spec/unit/lib/cloud_controller/backends/runners_spec.rb

```diff
@@ -344,10 +324,199 @@ module VCAP::CloudController
         staging_pending_app = dea_app1
         PackageModel.make(app: dea_app1.app, state: PackageModel::PENDING_STATE)
 
-        batch, _ = runners.dea_apps_hm9k(100, 0)
+        batch, _ = runners.dea_apps_hm9k
 
-        guids = batch.map { |entry| entry['id'] }
-        expect(guids).to include(staging_pending_app.guid)
+        found = false
+        batch.each do |item|
```

Comment:
This can be rewritten as:
```rb
expect(batch).to include(hash_including({
  'id' => staging_pending_app.guid,
  'package_state' => 'PENDING'
}))
```